tests/scalers_go/gcp_pubsub/gcp_pubsub_test.go

```diff
@@ -185,6 +188,7 @@ func TestScaler(t *testing.T) {
 	if sdkReady {
 		if createPubsub(t) == nil {
 			// test scaling
+			testDontScaleUpIfBelowThreshold(t, kc)
```

Comment:
we are naming this as `testActivation`, please follow the same convention to unify the tests